### PR TITLE
Show spouse name placeholder if spouse unverified

### PR DESCRIPTION
**Pull request by bengolder**

### CODE REVIEW

File: spec/models/intake_spec.rb [hunk 1]
```diff
@@ -375,6 +375,19 @@
...
+      let(:intake) do
+        create :intake, was_full_time_student: "yes", spouse_was_full_time_student: "yes"
+      end
+      before do
+        create :user, intake: intake, first_name: "Henrietta", last_name: "Huckleberry"
+      end
+
+      it "Shows a placeholder for the spouse name" do
+        expect(intake.student_names).to eq(["Henrietta Huckleberry", "Your spouse"])
+      end
```
Comment (tdooner) [inline spec/models/intake_spec.rb]: nit: spacing before `before` and the capitalized "Shows"

File: app/models/intake.rb [hunk 2]
```diff
@@ -245,10 +245,14 @@ def any_students?
       dependents.where(was_student: "yes" ).any?
   end
 
+  def spouse_name_or_placeholder
+    spouse.present? ? spouse.full_name : "Your spouse"
```
Comment (tdooner) [inline app/models/intake.rb]: fun rails tip: this exists to handle this pattern, should you so choose:

```ruby
spouse.full_name.presence || "Your Spouse"
```

Comment (bengolder) [reply app/models/intake.rb]: @tdooner that works for spouse presence or just full name presence?

Comment (bengolder) [reply app/models/intake.rb]: looks like that's for full_name. I changed it to

```ruby
spouse&.full_name || "Your spouse"
```

File: spec/models/intake_spec.rb (hunk 1 above)
Comment (bengolder) [reply spec/models/intake_spec.rb]: done, thanks!

File: app/models/intake.rb (hunk 2 above)
Comment (tdooner) [reply app/models/intake.rb]: Sorry yeah, I misread this. That looks good.

**Pull request merged**